common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -1276,18 +1290,12 @@
             *
             * @param {boolean} hideCaptions if `true` hides the caption,
             *     otherwise - show.
```

Comment:
Can we fix the docstring here to be more clear that this function is about transcripts, not captions? (If I understand correctly)